plugins/cxaudio/fu-cxaudio-firmware.c

```diff
@@ -0,0 +1,332 @@
+/*
+ * Copyright (C) 2005-2019 Synaptics Incorporated
+ * Copyright (C) 2019 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include "fu-common.h"
+
+#include "fu-cxaudio-firmware.h"
+
+struct _FuCxaudioFirmware {
+	FuSrecFirmwareClass	 parent_instance;
+	CXDEV_FILE_TYPE		 FileType;
+	CXDEV_DEVTYPE		 DeviceType;
+	CXDEV_EEPROM_CUSTOM_INFO cinfo;
+	gchar			*serial_number;
```

Comment:
Is it intentional that you're using CamelCase for some of these attributes?  Just seemed inconsistent.

Reply:
Not really, I used the same format from the sample code. I'll change to match the rest of the code.